sympy/polys/tests/test_gcd_hypothesis.py

```diff
@@ -0,0 +1,27 @@
+from hypothesis import given
+from hypothesis import strategies as st
+from sympy.abc import x
+from sympy.polys.polytools import Poly
+
+
+@given(
+    coefficients1=st.lists(st.integers()),
+    coefficients2=st.lists(st.integers()),
+    coefficients3=st.lists(st.integers()),
+)
+def test_gcd_hypothesis(coefficients1, coefficients2, coefficients3):
+    # instantiate polynomials
+    f = Poly(coefficients1, x, domain="ZZ")
+    g = Poly(coefficients2, x, domain="ZZ")
+    r = Poly(coefficients3, x, domain="ZZ")
+
+    # get gdc
```

Comment:
```suggestion
    # get gcd
```

Reply:
And anyway comments like this don't really add much. It's better to not include them. More useful comments would be comments that explain the mathematical properties that are being tested.